crawl-ref/source/dat/descript/de/monsters.txt

```diff
@@ -581,7 +581,7 @@ anaconda
 
 Eine sehr große Schlange, die sich mit erstaunlicher Geschwindigkeit bewegt. Sie
 tötet ihre Beute nicht durch Gift, sondern indem sie sie umschlingt und
-zerquetscht.
```

Comment:
In this sentence 'zerquetscht' works  better than 'erwürgt'. Umschlingen is the part that prevents movement. Zerquetschen (to crush) or Erwürgen (to strangle) do the damage. But erwürgen implies that only the throat is pressed shut. A big snake wraps around other parts of its victim too.

Reply:
OK. Thanks. I will revert it.

Reply:
done